main.tf

```diff
@@ -66,9 +79,19 @@ resource "aws_security_group_rule" "ingress" {
   cidr_blocks       = var.allowed_ip
   security_group_id = join("", aws_security_group.default.*.id)
 }
+resource "aws_security_group_rule" "ingress_ipv6" {
+  count = var.enable_security_group == true && local.enable_cidr_rules == true ? length(compact(var.allowed_ports)) : 0
```

Comment:
use other variables in place of 'enable_cidr_rules'

Reply:
Done